andela/Lit-Kigali1-Frontend
# #163519089 Set up empty react project

#### What does this PR do?
Sets up an empty react project
Adds webpack configuration
Adds babel configuration

#### Description of Task to be completed?
Set up of an empty react project

#### How should this be manually tested?
Clone the repo
Pull this branch
run `npm install`
run `npm run start`
navigate to: `localhost:8080` in your browser you will see a page welcoming you to Authors Haven

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#163519089

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
